book/src/pong-tutorial.md

```diff
@@ -1,13 +1,13 @@
 # Pong Tutorial
 
-To get a better feeling for how Amethyst works, we're going to implement a 
-Pong clone. You can find a [full Pong example][pong] (our end goal) in Amethyst's 
-examples folder. This tutorial breaks that project up into discrete steps so 
-it's easier to understand what everything is doing. If you've cloned the 
-Amethyst repo, you can run any of the examples like so:
+To get a better feeling for how Amethyst works, we're going to implement a
+Pong clone. You can find a [full Pong example][pong] (our end goal) in
+Amethyst's [examples] folder. This tutorial breaks that project up into discrete
+steps so it's easier to understand what everything is doing. If you've cloned
+the Amethyst repo, you can run any of the examples like so:
 
 ```norun
-cargo run --example pong_tutorial_01
+cargo run --example pong_tutorial_01 --features "vulkan"
```

Comment:
Could you please add that macOS users need to use `"metal"` instead of Vulkan?

Reply:
Sure, will do! :+1: